### DISCUSSION TITLE
[FR]Would like sample functions like tmuxinator

### DISCUSSION THREAD

**Discussion by AwesomeDog** [tmux-python/tmuxp#877] [Ideas]
When using https://github.com/tmuxinator/tmuxinator, one can start a new session by typing `tmuxinator new [project]`
and the command will open a sample config file located in `~/.tmuxinator/sample.yml`. One can simply start working by deleting and editing a few lines of the sample config.

The function can get rid of the chores of reading config manuals online, reduce debug time. Would be very nice to have.

**Comment by tony** [comment]
Idea: Fetching a starter config remotely

e.g. via git or a gist (similar to starters elsewhere, like [gatsby](https://www.gatsbyjs.com/docs/starters/)) and let the user verify / preview if it's secure

**Comment by tony** [comment]
Another idea: 

Custom default yaml config - for new configs, e.g. `~/.config/tmuxp/settings/default_config.yaml`

**Comment by tony** [comment]
Idea, allow multiple starter templates locally

- `~/.config/tmuxp/starters/basic.yaml`
- `~/.config/tmuxp/starters/3-panes.yaml`

```console
tmuxp new --starter basic
```

```console
tmuxp new --starter 3-panes
```

**Comment by AwesomeDog** [comment]
Idea: more layout template.

Some people use tmux like desktops: folders on the left, vim on the top right, real time logs on the bottom. And they might adjust the width and height by hand, which is a laborious work.

A few default layout templates might help, such as:
- IDE like layout
- System monitor layout 
etc.

Anther way that might help this scenario is "windowgram" borrowed from   https://github.com/oxidane/tmuxomatic#one-example
which can predefine a fine-tuned layout using ascii characters.
